src/xplugd.c

```diff
@@ -39,18 +39,20 @@ static char *rcfile(char *arg)
 	if (!arg)
 		arg = XPLUGRC;
 
-#ifdef GLOB_TILDES
```

Comment:
Gah, my bad.  Used GLOB_TILDES while testing the workaround on GLIBC. Thank you for fixing!